lib/modules/manager/maven-wrapper/artifacts.spec.ts

```diff
@@ -0,0 +1,156 @@
+import type { Stats } from 'fs';
```

Comment:
mock execution like in gradle wrapper

https://github.com/renovatebot/renovate/blob/37e5d01e0427b2cc72b36be4b921d66a3b31c0da/lib/modules/manager/gradle-wrapper/artifacts.spec.ts#L4